packages/backend/src/server/web/views/channel.pug

```diff
@@ -1,18 +1,20 @@
 extends ./base
 
 block vars
-	- const title = channel.name;
+	- const title = privateMode ? '非公開インスタンス' : channel.name;
 	- const url = `${config.url}/channels/${channel.id}`;
 
 block title
 	= `${title} | ${instanceName}`
```

Comment:
Including in the title whether the instance is private or not will not make much sense to the end users, so instead the title itself should be only the `instanceName`.

Reply:
That makes sense. I changed it to just be the instancename.